input/ig.xml

```diff
@@ -682,7 +682,7 @@
 				<reference value="Specimen/nasoswab"/>
 			</reference>
 			<name value="Specimen - Naso swab (COVID test)"/>
-			<description value="Naso swab specimen for COVID test&#xa;&#xa;Patient: Mr Li Wang"/>
+			<description value="This Specimen example is referenced by the *AU Core Pathology Result Observation* COVID test result example. &#xa;&#xa;Patient: Mr. Li Wang"/>
```

Comment:
Reformat. Shows a naso swab specimen referenced by the <insert hyperlink to the specific example> for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang.

Reply:
Due to markdown formatting half my comment disappeared. It was meant to be:
`Reformat. Shows a naso swab specimen referenced by the <insert hyperlink to the specific example> for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang.`

Please adjust now that I've made the full text of the comment visible.

Reply:
Applied